spec/presenters/submission_presenter_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe SubmissionPresenter do
-  subject { described_class.new(submission, rates) }
+  subject { described_class.new(submission, rates, submissions_repository) }
   let!(:rates) { submission.rates }
+  let!(:submissions_repository) { double }
```

Comment:
👍  for double
It would be good though to add some small test documenting why we need `submissions_repository` here.
You should not test results of the methods but e.g check if they are delegated to the repository successfully.
`expect(: submissions_repository).to receive(:next_to_rate)`